scripts/create_istio_system.sh

```diff
@@ -26,10 +26,17 @@ versions=$(kubectl get --ignore-not-found=true deploy istiod -n $ISTIO_NAMESPACE
 if [[ $versions == "" ]]; then
   echo "running istioctl install"
   istioctl install -i $ISTIO_NAMESPACE -y ${ISTIO_FILES[@]/#/-f }
-
+  if [[ $? -ne 0 ]]; then
```

Comment:
Could we do

```suggestion
  if ! istioctl install -i $ISTIO_NAMESPACE -y ${ISTIO_FILES[@]/#/-f }; then
```

Reply:
same for the other change

Reply:
I updated this in the two places you recommended - we were also using a similar check for the istioctl analyze call so I went ahead and updated it there for consistency in the script.